lib/hardware/led_light.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef PET_PASSAGE_HARDWARE_LED_LIGHT_H_
+#define PET_PASSAGE_HARDWARE_LED_LIGHT_H_
+
+#include "Arduino.h"
+#include "led_base.hpp"
+
+template <uint8_t Pin>
+class LedLight : public LedBase {
+ public:
```

Comment:
There are only public mebers thus this should be declared as a stuct not a class.